### PR TITLE
Support for BBC Reith Qalam - font & font family

### LINKED ISSUE #3802: Add Reith Qalam font & font family
**Is your feature request related to a problem? Please describe.**
Add BBC Reith Qalan font and font family

**Describe the solution you'd like**
- Font weights are `F_QALAM_REGULAR` and `F_QALAM_BOLD`
- Add Reith Qalam including link to font location on ws-downloads domain to https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/fonts.js
- Add Reith Qalam font & font family, plus fallbacks to https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/font-families.js
- This should be a major version bump of the `psammead-styles` package, to prevent automated dependency updates 

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves #3802 

**Overall change:** 
Adding the font and font family for BBC Reith Qalam

**Code changes:**
- Adding BBC Reith Qalam fonts for Arabic, Pashto, Persian and Urdu
- Removal of BBC Nassim fonts for Arabic, Pashto, Persian and Urdu

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by rhenshaw56 — APPROVED** [review]

File: packages/utilities/psammead-styles/src/__snapshots__/fonts.test.js.snap [hunk 1]
```diff
@@ -46,6 +81,17 @@ exports[`Psammead Styles - Fonts should match LATHA REGULAR base font url 1`] =
   }"
 `;
 
+exports[`Psammead Styles - Fonts should match LATHA REGULAR with overridden url 1`] = `
```
Comment (simonsinclair) [inline packages/utilities/psammead-styles/src/__snapshots__/fonts.test.js.snap:84]: I may be missing something in the diff, but would you mind clarifying what's caused these block additions for fonts unrelated to Qalam?

**Review by simonsinclair — APPROVED** [review]
This is super, Karina. 👍

File: packages/utilities/psammead-styles/src/__snapshots__/fonts.test.js.snap (hunk 1 above)
Comment (karinathomasbbc) [reply packages/utilities/psammead-styles/src/__snapshots__/fonts.test.js.snap:84]: I updated the test to have a better desctription in the snapshot https://github.com/bbc/psammead/blob/af4c0b40910e7a7e765c9ebd24d782dc3f9827ab/packages/utilities/psammead-styles/src/fonts.test.js#L16

Comment (karinathomasbbc) [reply packages/utilities/psammead-styles/src/__snapshots__/fonts.test.js.snap:84]: Previously it was https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/fonts.test.js#L16

Comment (simonsinclair) [reply packages/utilities/psammead-styles/src/__snapshots__/fonts.test.js.snap:84]: Aha! Thanks.

**Pull request merged**